doc/developer/testdrive.md

```diff
@@ -42,7 +42,7 @@ Testdrive is the preferred system test framework when testing:
 
 `testdrive` is currently sub-optimal for the situations listed below and `sqllogictest` is the better driver in those cases:
 - test cases containing many `EXPLAIN` statements and a lot of recorded query plans
-- tests that require the use of `INSERT SELECT, `UPDATE` and `DELETE`
+- tests that require the use of `INSERT`, `SELECT`, `UPDATE`, and `DELETE`
```

Comment:
It should have been `INSERT SELECT`, as a single thing . Anyway, all of those statements are natively supported by Mz so there is no need for sqllogictest to be used to proxy them through Postgres. I will submit a followup PR to that effect.